src/example-types/metricGroups/pivot.js

```diff
@@ -151,22 +151,22 @@ let buildQuery = async (node, schema, getStats) => {
   return query
 }
 
-// TODO: instead of groupN, maybe look at query to say something more valuable?
-//  e.g. => node.groups[n].field + ' ' + node.groups[n].type (aka 'Organization.NameState fieldValues'), maybe customized per type?
+// TODO: instead of rowN, maybe look at query to say something more valuable?
```

Comment:
This also doesn’t make sense, because this is actually referring to column names. We might want to just drop pivot flattening entirely since I don’t think it supports columns at the moment.